src/server/pfs/server/driver.go

```diff
@@ -684,16 +667,20 @@ func (d *driver) aliasCommit(txnCtx *txncontext.TransactionContext, parent *pfs.
 			parentCommitInfo.ChildCommits = append(parentCommitInfo.ChildCommits, commit)
 			return nil
 		}); err != nil {
+			if col.IsErrNotFound(err) {
+				return nil, pfsserver.ErrCommitNotFound{parent}
```

Comment:
Will this error have a stack?

Reply:
Yeah, this won't (but this is how PFS returns all `pfsserver` errors) - I can do a pass on these in a separate PR if that's ok.